2019-09-30T18:11:40.2767890-05:00 0HLQ5VE6FDTU3:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:11:40.3425310-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:11:40.3795850-05:00 0HLQ5VE6FDTU3:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:11:40.3968020-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:11:40.4046210-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executed action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 0.3404ms. (50a9e262)
2019-09-30T18:11:40.4168570-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executing ObjectResult, writing value of type '"System.String[]"'. (8a1b66c8)
2019-09-30T18:11:40.5501430-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 163.7062ms (afa2e885)
2019-09-30T18:11:40.5539610-05:00 0HLQ5VE6FDTU3:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:11:40.5623090-05:00 0HLQ5VE6FDTU3:00000001 [INF] Request finished in 292.9936ms 200 application/json; charset=utf-8 (791a596a)
2019-09-30T18:11:42.5398210-05:00 0HLQ5VE6FDTU4:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:11:42.5449390-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:11:42.5498180-05:00 0HLQ5VE6FDTU4:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:11:42.5550560-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:11:42.5801190-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executed action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)", returned result "Microsoft.AspNetCore.Mvc.ObjectResult" in 0.0827ms. (50a9e262)
2019-09-30T18:11:42.5843040-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executing ObjectResult, writing value of type '"System.String[]"'. (8a1b66c8)
2019-09-30T18:11:42.5892760-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 35.1071ms (afa2e885)
2019-09-30T18:11:42.5932490-05:00 0HLQ5VE6FDTU4:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:11:42.5974980-05:00 0HLQ5VE6FDTU4:00000001 [INF] Request finished in 70.4324ms 200 application/json; charset=utf-8 (791a596a)
2019-09-30T18:11:59.4467120-05:00 0HLQ5VE6FDTU4:00000002 [INF] Request starting HTTP/1.1 GET https://localhost:5001/swagger/index.html   (ca22a1cb)
2019-09-30T18:11:59.5593530-05:00 0HLQ5VE6FDTU4:00000002 [INF] Request finished in 134.5495ms 200 text/html (791a596a)
2019-09-30T18:11:59.6745450-05:00 0HLQ5VE6FDTU4:00000003 [INF] Request starting HTTP/1.1 GET https://localhost:5001/swagger/swagger-ui.css   (ca22a1cb)
2019-09-30T18:11:59.7025470-05:00 0HLQ5VE6FDTU6:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/swagger/swagger-ui-bundle.js   (ca22a1cb)
2019-09-30T18:11:59.8166890-05:00 0HLQ5VE6FDTU7:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/swagger/swagger-ui-standalone-preset.js   (ca22a1cb)
2019-09-30T18:11:59.8409560-05:00 0HLQ5VE6FDTU4:00000003 [INF] Sending file. Request path: '"/swagger-ui.css"'. Physical path: '"N/A"' (27b0a520)
2019-09-30T18:11:59.8443340-05:00 0HLQ5VE6FDTU4:00000003 [INF] Request finished in 171.9859ms 200 text/css (791a596a)
2019-09-30T18:11:59.8566820-05:00 0HLQ5VE6FDTU7:00000001 [INF] Sending file. Request path: '"/swagger-ui-standalone-preset.js"'. Physical path: '"N/A"' (27b0a520)
2019-09-30T18:11:59.8602060-05:00 0HLQ5VE6FDTU7:00000001 [INF] Request finished in 43.9857ms 200 application/javascript (791a596a)
2019-09-30T18:11:59.8742610-05:00 0HLQ5VE6FDTU6:00000001 [INF] Sending file. Request path: '"/swagger-ui-bundle.js"'. Physical path: '"N/A"' (27b0a520)
2019-09-30T18:11:59.8773210-05:00 0HLQ5VE6FDTU6:00000001 [INF] Request finished in 195.6966ms 200 application/javascript (791a596a)
2019-09-30T18:12:00.3743150-05:00 0HLQ5VE6FDTU6:00000002 [INF] Request starting HTTP/1.1 GET https://localhost:5001/swagger/v1/swagger.json   (ca22a1cb)
2019-09-30T18:12:00.5183380-05:00 0HLQ5VE6FDTU6:00000002 [INF] Request finished in 165.1309ms 200 application/json;charset=utf-8 (791a596a)
2019-09-30T18:15:41.7400890-05:00 0HLQ5VGEEAU89:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:15:41.7989370-05:00 0HLQ5VGEEAU89:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:15:41.8344000-05:00 0HLQ5VGEEAU89:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:15:41.8488940-05:00 0HLQ5VGEEAU89:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:15:42.8813240-05:00 0HLQ5VGEEAU89:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 1038.7673ms (afa2e885)
2019-09-30T18:15:42.8855380-05:00 0HLQ5VGEEAU89:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:15:42.9567930-05:00 0HLQ5VGEEAU89:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at QuoteApi.Controllers.ValuesController.Get() in /Users/nbontha/dev/dockertests/QuoteApi/QuoteApi/Controllers/ValuesController.cs:line 20
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2019-09-30T18:15:43.0366110-05:00 0HLQ5VGEEAU89:00000001 [INF] Request finished in 1301.5806ms 500 text/html; charset=utf-8 (791a596a)
2019-09-30T18:15:44.6201930-05:00 0HLQ5VGEEAU8A:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:15:44.6265760-05:00 0HLQ5VGEEAU8A:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:15:44.6316930-05:00 0HLQ5VGEEAU8A:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:15:44.6359190-05:00 0HLQ5VGEEAU8A:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:15:44.8907160-05:00 0HLQ5VGEEAU8A:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 256.089ms (afa2e885)
2019-09-30T18:15:44.8939380-05:00 0HLQ5VGEEAU8A:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:15:44.8995460-05:00 0HLQ5VGEEAU8A:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at QuoteApi.Controllers.ValuesController.Get() in /Users/nbontha/dev/dockertests/QuoteApi/QuoteApi/Controllers/ValuesController.cs:line 20
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2019-09-30T18:15:44.9115380-05:00 0HLQ5VGEEAU8A:00000001 [INF] Request finished in 295.9773ms 500 text/html; charset=utf-8 (791a596a)
2019-09-30T18:17:04.2216210-05:00 0HLQ5VH712PR3:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:17:04.2753260-05:00 0HLQ5VH712PR3:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:17:04.3142920-05:00 0HLQ5VH712PR3:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:17:04.3321170-05:00 0HLQ5VH712PR3:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:17:05.0057060-05:00 0HLQ5VH712PR3:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 684.9858ms (afa2e885)
2019-09-30T18:17:05.0091810-05:00 0HLQ5VH712PR3:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:17:05.0614610-05:00 0HLQ5VH712PR3:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at QuoteApi.Controllers.ValuesController.Get() in /Users/nbontha/dev/dockertests/QuoteApi/QuoteApi/Controllers/ValuesController.cs:line 20
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2019-09-30T18:17:05.1512730-05:00 0HLQ5VH712PR3:00000001 [INF] Request finished in 936.5393ms 500 text/html; charset=utf-8 (791a596a)
2019-09-30T18:17:06.3098590-05:00 0HLQ5VH712PR4:00000001 [INF] Request starting HTTP/1.1 GET https://localhost:5001/api/values   (ca22a1cb)
2019-09-30T18:17:06.3140110-05:00 0HLQ5VH712PR4:00000001 [INF] Executing endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (500cc934)
2019-09-30T18:17:06.3188890-05:00 0HLQ5VH712PR4:00000001 [INF] Route matched with "{action = \"Get\", controller = \"Values\"}". Executing controller action with signature "Microsoft.AspNetCore.Mvc.ActionResult`1[System.Collections.Generic.IEnumerable`1[System.String]] Get()" on controller "QuoteApi.Controllers.ValuesController" ("QuoteApi"). (122b2fdf)
2019-09-30T18:17:06.3648400-05:00 0HLQ5VH712PR4:00000001 [INF] Executing action method "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" - Validation state: Valid (dad538d7)
2019-09-30T18:17:06.6656820-05:00 0HLQ5VH712PR4:00000001 [INF] Executed action "QuoteApi.Controllers.ValuesController.Get (QuoteApi)" in 322.3933ms (afa2e885)
2019-09-30T18:17:06.6688590-05:00 0HLQ5VH712PR4:00000001 [INF] Executed endpoint '"QuoteApi.Controllers.ValuesController.Get (QuoteApi)"' (99874f2b)
2019-09-30T18:17:06.6744050-05:00 0HLQ5VH712PR4:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.NullReferenceException: Object reference not set to an instance of an object.
   at QuoteApi.Controllers.ValuesController.Get() in /Users/nbontha/dev/dockertests/QuoteApi/QuoteApi/Controllers/ValuesController.cs:line 20
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Routing.EndpointRoutingMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2019-09-30T18:17:06.6868410-05:00 0HLQ5VH712PR4:00000001 [INF] Request finished in 376.9479ms 500 text/html; charset=utf-8 (791a596a)
2019-09-30T18:26:12.4834790-05:00  [INF] Entity Framework Core "2.2.6-servicing-10079" initialized '"QuoteDbContext"' using provider '"Microsoft.EntityFrameworkCore.SqlServer"' with options: "None" (9958f5bb)
2019-09-30T18:26:28.6612610-05:00  [FTL] Application startup exception (6c57a5ba)
System.AggregateException: One or more errors occurred. (A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 40 - Could not open a connection to SQL Server)) ---> System.Data.SqlClient.SqlException: A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 40 - Could not open a connection to SQL Server)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Migrations.HistoryRepository.ExistsAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.MigrateAsync(String targetMigration, CancellationToken cancellationToken)
   at Quote.DataAccess.DbLayers.DbInitializer.SeedAsync() in /Users/nbontha/dev/dockertests/QuoteApi/Quote.DataAccess/DbLayers/IDbInitializer.cs:line 27
   --- End of inner exception stack trace ---
   at QuoteApi.Startup.Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory, IDbInitializer databaseInitializer) in /Users/nbontha/dev/dockertests/QuoteApi/QuoteApi/Startup.cs:line 85
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Hosting.ConventionBasedStartup.Configure(IApplicationBuilder app)
   at Microsoft.AspNetCore.Mvc.Internal.MiddlewareFilterBuilderStartupFilter.<>c__DisplayClass0_0.<Configure>g__MiddlewareFilterBuilder|0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.HostFilteringStartupFilter.<>c__DisplayClass0_0.<Configure>b__0(IApplicationBuilder app)
   at Microsoft.AspNetCore.Hosting.Internal.AutoRequestServicesStartupFilter.<>c__DisplayClass0_0.<Configure>b__0(IApplicationBuilder builder)
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.BuildApplication()
   at Microsoft.AspNetCore.Hosting.Internal.WebHost.StartAsync(CancellationToken cancellationToken)
   at Microsoft.AspNetCore.Hosting.WebHostExtensions.RunAsync(IWebHost host, CancellationToken token, String shutdownMessage)
---> (Inner Exception #0) System.Data.SqlClient.SqlException (0x80131904): A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: TCP Provider, error: 40 - Could not open a connection to SQL Server)
   at System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, Boolean applyTransientFaultHandling, String accessToken)
   at System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   at System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   at System.Data.ProviderBase.DbConnectionPool.WaitForPendingOpen()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenDbConnectionAsync(Boolean errorsExpected, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.OpenAsync(CancellationToken cancellationToken, Boolean errorsExpected)
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerDatabaseCreator.<>c__DisplayClass20_0.<<ExistsAsync>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.EntityFrameworkCore.SqlServer.Storage.Internal.SqlServerExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Migrations.HistoryRepository.ExistsAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Migrations.Internal.Migrator.MigrateAsync(String targetMigration, CancellationToken cancellationToken)
   at Quote.DataAccess.DbLayers.DbInitializer.SeedAsync() in /Users/nbontha/dev/dockertests/QuoteApi/Quote.DataAccess/DbLayers/IDbInitializer.cs:line 27
ClientConnectionId:00000000-0000-0000-0000-000000000000<---

